pallets/staking-rewards/src/tests.rs

```diff
@@ -417,16 +417,18 @@ fn nominators_rewards_disbursement_works() {
                 + initial_treasury_balance
                 + signer_balance
                 + initial_rewards_pool_balance
-                + ENDOWMENT // NOM_1_STASH
-                + EXISTENTIAL_DEPOSIT // added to the rewards pool to ensure pool existence
+                // NOM_1_STASH
+                + ENDOWMENT
+                // added to the rewards and the rent pools to ensure pool existence
+                + 2 * EXISTENTIAL_DEPOSIT
         );
         assert_eq!(initial_rewards_pool_balance, pool_balance);
         assert_eq!(
             initial_total_stakeable,
             (non_stakeable.left_from_one() * initial_total_issuance).saturating_sub(pool_balance)
         );
 
-        let era_duration_in_millis = era_duration * MILLISECS_PER_BLOCK;
+        let era_duration_in_millis = era_duration * u64::from(MILLISECS_PER_BLOCK);
```

Comment:
why have you changed it to u32 than?

Reply:
since it should be of `<T as pallet_timestamp::Config>::Moment` type but yeah, looks a bit weird. Will restore the type